<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="G921LTRV" failures="1" tests="1" name="TestNg" time="0.06" errors="0" timestamp="2024-07-12T09:42:36 IST">
  <testcase classname="testNgDemo.testng.SoftAssertEX" name="softAssert" time="0.011">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [11] but found [10],
	expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [11] but found [10],
expected [true] but found [false]
at testNgDemo.testng.SoftAssertEX.softAssert(SoftAssertEX.java:14)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:65)
at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:105)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- softAssert -->
</testsuite> <!-- TestNg -->
